### ISSUE TITLE
[Feature] WCAG 1.2 Failures

### ISSUE THREAD

**Issue by dustinwhisman** [dustinwhisman/inaccessible#4]
## Description

Add a test page that includes as many failures for this WCAG Guideline as possible alongside successful implementations so both false positives and false negatives can be caught when evaluating accessibility tools.
